lib/routes.js

```diff
@@ -29,7 +29,13 @@ module.exports = (contract) => {
     return {
       id: _.kebabCase(doc.title),
       type: 'article',
-      content: doc
+      content: doc,
+      sections: doc.data.filter((entry) => { return entry.includes('header') && entry[1].level === 2 }).map((entry) => {
```

Comment:
Lets make the header extraction logic part of the DocViewer component, within the `.variant()` function

Reply:
@jviotti 	I did it this way to avoid passing to the view all the article content. Imagine every page that needs only the metadata of the article/doc, and yet we include the whole markdown file for every document.

Reply:
But the DocViewer component needs access to the whole markdown file anyway, right?

Reply:
@jviotti 	It needs access for the document it renders, not all the other documents. We just need the metadata (path, title) for the other elements in the table of contents.

Reply:
Ahhhh I get it now. OK, sounds good